src/sgame/sg_bot_nav.cpp

```diff
@@ -628,6 +628,7 @@ void BotMoveToGoal( gentity_t *self )
 {
 	vec3_t dir;
 	VectorCopy( self->botMind->nav.dir, dir );
+	const auto& state = self->client->ps;
```

Comment:
The rest of the code uses this convention:
```suggestion
	const playerState_t& ps = self->client->ps;
```